python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/aaa_accounting.schema.yml

```diff
@@ -43,6 +45,7 @@ keys:
               type:
                 type: str
                 valid_values: ["none", "start-stop", "stop-only"]
+                required: true
```

Comment:
I would prefer not to change the schema now, but just update the template to check for the key. In general we should not have required keys in eos_cli_config_gen (except primary keys) since it breaks validation of custom structured configuration.